### ISSUE TITLE
Make empty queues act like silent queues

### ISSUE THREAD

**Issue by xurizaemon** [CiviCooP/org.civicoop.civirules#109]
We had a site using CiviRules see a spike in information written to the CiviCRM debug log when using CiviRules. CiviRules was reporting repeatedly,

> $not ready for release = CRM_Queue_DAO_QueueItem Object (/* much information follows */)

which I believe was reporting, "queue item is scheduled for a future date, skipping".

IMO this case - and the fact that a queue is empty - are not "remarkable" enough to debug, so here's a PR to remove that debug code. (If this was an unexpected state, or one was debugging whether the queue mechanism works, it makes sense to see it. If the queue is operating normally, IMO it should be silent.)

Depending on how stealItem() is expected to be used, you MIGHT want to hear about a failed attempt, I'm not sure.

(ping @joemcl)

**Comment by joemcl** [comment]
Thanks @xurizaemon I'll update our install of CiviRules and test it out again once this PR is merged.

**Comment by xurizaemon** [comment]
For context - the site in question produced 660MB in a day of "debug log" for whatever was in their queue at the time, because the "much information follows" above contains a printing of the queue item. Let's say it set off some bells :)

**Comment by xurizaemon** [comment]
@joemcl - you can test it now, see email

**Comment by jaapjansma** [comment]
Thanks @xurizaemon and @joemcl the PR is merged into the master branch.

**Comment by xurizaemon** [comment]
@totten - I wonder if CiviCRM's queue implementation (which shared this same debug) would benefit from same treatment? Is it useful information there?
